Content.Server/Dragon/DragonSystem.Rule.cs

```diff
@@ -27,43 +27,27 @@ private int RiftsMet(DragonComponent component)
         return finished;
     }
 
-    protected override void Started(EntityUid uid, DragonRuleComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
-    {
-        base.Started(uid, component, gameRule, args);
-
-        var eligible = EntityQuery<StationEventEligibleComponent>().Select(x => x.Owner).ToList();
-
-        if (!eligible.Any())
-            return;
-
-        var station = _random.Pick(eligible);
-
-        if (_station.GetLargestGrid(EntityManager.GetComponent<StationDataComponent>(station)) is not { } grid)
-            return;
-
-        Spawn("MobDragon", Transform(grid).MapPosition);
-    }
-
     private void OnRiftRoundEnd(RoundEndTextAppendEvent args)
     {
-        var dragons = EntityQuery<DragonComponent>(true).ToList();
+        var dragons = EntityQuery<DragonComponent>().ToList();
```

Comment:
Use Count<DragonComponent> if only the count is needed, much faster.

Reply:
didnt know that exist